x-pack/plugins/ml/public/embeddables/common/ml_job_additional_settings.tsx

```diff
@@ -0,0 +1,477 @@
+                  value={bucketSpan}
+                  onChange={(e) => {
+                    setBucketSpan(e.target.value);
+                    setState(STATE.VALIDATING);
+                  }}
+                />
+              </EuiFormRow>
+              <EuiSpacer size="l" />
+              <EuiFormRow>
+                <EuiCheckbox
+                  id="startJob"
+                  data-test-subj={`mlLensLayerStartJobCheckbox_${layerIndex}`}
+                  checked={startJob}
+                  onChange={(e) => setStartJobWrapper(e.target.checked)}
+                  label={i18n.translate(
+                    'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.start',
+                    {
+                      defaultMessage: 'Start the job after saving',
+                    }
+                  )}
+                />
+              </EuiFormRow>
+
+              <EuiSpacer size="s" />
+              <EuiFormRow>
+                <EuiCheckbox
+                  id="realTime"
+                  disabled={startJob === false}
+                  data-test-subj={`mlLensLayerRealTimeCheckbox_${layerIndex}`}
+                  checked={runInRealTime}
+                  onChange={(e) => setRunInRealTime(e.target.checked)}
+                  label={i18n.translate(
+                    'xpack.ml.embeddables.lensLayerFlyout.createJobCallout.additionalSettings.realTime',
+                    {
+                      defaultMessage: 'Leave the job running for new data',
+                    }
+                  )}
+                />
+              </EuiFormRow>
+              <EuiSpacer size="m" />
+            </EuiAccordion>
+          </EuiForm>
+          <EuiSpacer size="m" />
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiButton
+                disabled={
+                  state === STATE.VALIDATING ||
+                  jobId === '' ||
+                  jobIdValidationError !== '' ||
+                  bucketSpanValidationError !== ''
+                }
+                onClick={createJob.bind(null, layerIndex)}
+                size="s"
+                data-test-subj={`mlLensLayerCreateJobButton_${layerIndex}`}
+              >
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.createJobButton.saving"
+                  defaultMessage="Create job"
+                />
+              </EuiButton>
+            </EuiFlexItem>
+
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                onClick={createADJobInWizard.bind(null, layerIndex)}
+                size="s"
+                iconType="popout"
+                iconSide="right"
+                data-test-subj={`mlLensLayerCreateWithWizardButton_${layerIndex}`}
+              >
+                <FormattedMessage
+                  id="xpack.ml.embeddables.lensLayerFlyout.createJobButton"
+                  defaultMessage="Create job using wizard"
```

Comment:
When I use this link from the Map flyout it takes me only to the first 'Select data view or saved search' step of the wizard, where I was expecting it would take me straight into the Geo wizard, in a similar way for metric jobs from Dashboard which takes you into the Single / Multi-metric wizard. Is it possible to go directly into the Geo wizard with the geo field populated?